### PR TITLE
revert swarmm change

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**: `swarmm` differentiates itself as "SwarmMode" compared to "Swarm" for `swarm` 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
revert swarmm change
```

### CODE REVIEW

File: parts/swarm/swarmmastervars.t [hunk 1]
```diff
@@ -97,7 +97,7 @@
       ]
     ],
 {{if .OrchestratorProfile.IsSwarmMode}}
-    "orchestratorName": "swarm",
+    "orchestratorName": "swarmm",
```
Comment (CecileRobertMichon) [inline parts/swarm/swarmmastervars.t]: this is only used to name resources, can we make it swarrmmode for clarity?

Comment (CecileRobertMichon) [reply parts/swarm/swarmmastervars.t]: "swarmm" just looks like a spelling mistake which is what tricked you into changing it the first time

Comment (jackfrancis) [reply parts/swarm/swarmmastervars.t]: Preference for not making a change

**Pull request merged**